R/vcf2bedpe.R

```diff
@@ -0,0 +1,163 @@
+# vcf2bedpe
+
+vcf2bedpe <- function(x, filename = NULL, header = FALSE) {
+  catv('CONVERT VCF TO BEDPE\n')
+  if (!is.null(attr(x, 'vcf')) && attr(x, 'vcf') && all(names(x) == c('header','vcf'))) {
+    x <- x$vcf;
+  }
+  else {
+    catv(' * This is not an vcf!\n')
+    stop();
+  }
+  # use UCSC chromosome convention and subtract one position for start
+  x$CHROM <- gsub('chr', '', x$CHROM);
+  x$POS <- x$POS - 1;
+  if (!'SVTYPE' %in% names(x)) {
+    stop('SVTYPE column must exist in VCF file');
+  }
```

Comment:
Is this function designed for vcfs containing SVs only? If that is the case, should we not consider embedding this property in the function name itself which currently looks like a generic vcf processor?

Reply:
My understanding is that we use BEDPE to represent SVs and not simple variants so you wouldn't really use it for VCFs that aren't generated by SV callers.